Mathlib/CategoryTheory/Sites/CompatibleSheafification.lean

```diff
@@ -91,10 +91,10 @@ the sheafification of `P ⋙ F`, functorially in `P`. -/
 noncomputable def sheafificationWhiskerRightIso :
     J.sheafification D ⋙ (whiskeringRight _ _ _).obj F ≅
       (whiskeringRight _ _ _).obj F ⋙ J.sheafification E := by
-  refine' Functor.associator _ _ _ ≪≫ _
-  refine' isoWhiskerLeft (J.plusFunctor D) (J.plusFunctorWhiskerRightIso _) ≪≫ _
-  refine' _ ≪≫ Functor.associator _ _ _
-  refine' (Functor.associator _ _ _).symm ≪≫ _
+  refine Functor.associator _ _ _ ≪≫ ?_
```

Comment:
Again

Reply:
I didn't change arrows in at least _some_ of these files…